packages/block-library/src/image/edit.js

```diff
@@ -168,6 +169,25 @@ export function ImageEdit( {
 		setTemporaryURL( undefined );
 	}
 
+	// TODO: If this would be the way to go, we need to handle similarly audio and video.
+	function onHTMLDrop( HTML ) {
+		const blocks = pasteHandler( { HTML, mode: 'BLOCKS' } );
```

Comment:
Does this come from inside the editor? If so, we shouldn't use paste handler, we should just parse the blocks?